pycm/pycm_util.py

```diff
@@ -435,3 +435,65 @@ def matrix_combine(matrix_1, matrix_2):
             temp_dict[class_2] = tmp
         result_matrix[class_1] = temp_dict
     return result_matrix
+
+
+def add_number_label(ax, classes, matrix, cmap, number_label_cmap):
+    """
+    Add Number labels to Confusion matrix plot.
+
+    :param ax: Confusion matrix axes
+    :type ax: matplotlib.axes
+    :param classes: classes of matrix
+    :type classes: list
+    :param matrix: derived matrix of Confusion matrix
+    :type matrix: numpy.array
+    :param cmap: color map
+    :type cmap: matplotlib.colors.ListedColormap
+    :param number_label_cmap: number label cmap
+    :type number_label_cmap: matplotlib.colors.ListedColormap
+    :return: none
+    """
+    for i in range(len(classes)):
+        for j in range(len(classes)):
+            color = cmap(1 - (float(matrix[i][j]) / float(matrix.max())))
```

Comment:
This method for text color selection is not accurate enough.
Please check this :
```python
diff_matrix = float(matrix.max()) - matrix
diff_matrix_max = float(diff_matrix.max())
for i in range(len(classes)):
    for j in range(len(classes)):
        color = cmap(round(diff_matrix[i][j]/diff_matrix_max))
```

Reply:
Thank you for this.
I've tried this approach without that `round` and it seems it's working better.

Reply:
Are you sure it's better ?
Please take a look at this example, and run it with/without `round` function :
```python
import numpy as np
from pycm import *
np.random.seed(100)
x1 = np.random.randint(low=0,high=3,size=20000)
x2 = np.random.randint(low=0,high=3,size=20000)
cm1 = ConfusionMatrix(x1,x2)
```

Reply:
Here is the result:
+ Confusion_Matrix_without_round
![Confusion_Matrix_without_round](https://user-images.githubusercontent.com/43045767/95049979-a7647700-06f7-11eb-937e-7beeb1701b2b.png)
+ Confusion_Matrix_with_round
![Confusion_Matrix_with_round](https://user-images.githubusercontent.com/43045767/95049984-a92e3a80-06f7-11eb-9037-c7858eb6398a.png)
As you can see using `round` make them all white.
Is it true?

Reply:
With `round` result is better.
Please test this :
```python
diff_matrix = float(matrix.max()) - matrix
diff_matrix_max = float(diff_matrix.max())
for i in range(len(classes)):
    for j in range(len(classes)):
        color_index = float(round(diff_matrix[i][j]/diff_matrix_max))
        color = cmap(color_index)
```

Reply:
Thank you it is much better now:
![Confusion_Matrix_(Normalized)](https://user-images.githubusercontent.com/43045767/95161302-a55def00-07af-11eb-8ec4-df25bec7c347.png)